core/cat/looking_glass/stray_cat.py

```diff
@@ -35,6 +35,8 @@ def __init__(
         ws: WebSocket = None,
     ):
         self.__user_id = user_id
+        self.__user_data = user_data
```

Comment:
why do you still need `__user_id`? It can be just a property, grabbing the user id from user_data, isn't it?

Reply:
The purpose of this pull request is to be a bug fix and to minimise Breaking changes.

IMHO
in my opinion you are right, but I would approach it first by no longer passing the constructor user_id and populating the property from user_data or perhaps even leaving user_data alone and calling user_data.id. 
But this is a broader reasoning that might be worth another PR